apps/myaccount/src/models/global.ts

```diff
@@ -26,4 +26,5 @@ export interface GlobalInterface {
     isApplicationsPageVisible: boolean;
     isGlobalLoaderVisible: boolean;
     supportedI18nLanguages: SupportedLanguagesMeta;
+    openAction: string;
```

Comment:
Should we rethink the term openAction? I feel like this term is a bit misleading given that we use it to decide the currently editing section.

Reply:
Fixed by 26e3ed87